packer/http/mint-19.seed

```diff
@@ -0,0 +1,51 @@
+#### Contents of the preconfiguration file
```

Comment:
what is the use of this file?

Reply:
A seed file should help with the automatic installation of Debian/Ubuntu/LinuxMint, as fas as I know, so that you don't have to give answers manually while you perform the installation.
But I am not sure whether this file does its job properly. Hopefully Deepanshu has tested it.

Reply:
Yes you are right. I got this file from tutorials at linux mint website - https://community.linuxmint.com/tutorial/view/54

I had to edit some of the parts. Challenging part was to get `openssh-server` installed and running(Since there are no good tutorials on this). But that is also done now. Earlier I was testing with a local iso image. Now testing with a fresh install on different machine. I found some issues and fixed them. Now it works perfectly. 

 Will soon upload the created `linux-mint-xfce-32bit` box file to vagrant cloud as version `1.0.3`